test/testenv/verificationutils.go

```diff
@@ -272,3 +272,11 @@ func VerifyIndexExistsOnS3(deployment *Deployment, podName string, indexName str
 		return indexFound
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(true))
 }
+
+//VerifyMCConfiguredWithLM Verify that MC is configured with License Master
```

Comment:
verifies vs verify?

Reply:
Just followed the syntax followed for other functions.